docs/testing/e2e-tests.md

```diff
@@ -0,0 +1,7 @@
+---
+title: End-to-End Tests
+---
+
+The E2E test suite can be run against a properly bootstrapped cluster. Please refer to [this repo](https://github.com/redhat-appstudio/e2e-tests) for details on how to build and run the tests.
```

Comment:
How do we view the e2e tests? When are they run? When do we need to add/remove tests from the suite?

Reply:
I mentioned when it runs on the README for the CI. The same file also has a link to the dashboard that shows the jobs. I think that this section should be improved and explain how to run the e2e test in a development environment, but it's out of the scope of this PR.